deploy/db/sqlite_schema.sql

```diff
@@ -7,6 +7,8 @@ CREATE TABLE IF NOT EXISTS tokens (
   refresh_token BYTEA       NOT NULL,
   token_expiry  BIGINT      NOT NULL,
   disconnected  BOOLEAN     NOT NULL DEFAULT 0,
+  aith_type     VARCHAR(255) DEFAULT "OAuth2",
```

Comment:
Typo `aith_type`

Reply:
Ah, good spot.. more reason to merge the migrator into the proxy and be done with this.